### DISCUSSION TITLE
Modal form with Two inputs with one label

### DISCUSSION THREAD

**Discussion by lgalloherran** [twilio-labs/paste#2701] [Help: Figma] [answered]
For the Toll-free Verification project, I'm working on a modal form and Product requested the 'phone number' field to have an input for the ISO (country), and another input for the phone number. I proposed this design and I wanted to pick your brains if this is a good approach given that adding a new input with label will imply redesigning this modal page and we currently don't have the time to do it. 

<img width="683" alt="Captura de Pantalla 2022-09-28 a la(s) 4 38 01 p m" src="https://user-images.githubusercontent.com/72419222/192895152-3345d5c3-28ba-4777-b509-e788566939f5.png">

**Comment by kelseycavitt** [comment, marked as answer]
Hi Luz! 
We think that, if possible, it'd be best to still have labels for each of those inputs to be as clear as possible about what input next to the ISO will be. Our suggestion is to change the "Phone number" input to say "Country", then the next input to be "Phone number", see the pic attached. This seems to be what happens elsewhere in the product. 

![Luz phone number input example](https://user-images.githubusercontent.com/1484014/193709418-acce46e8-5747-4fa7-833f-2ceb3509dafb.jpg)

Also, check out this related GH discussion topic: https://github.com/twilio-labs/paste/discussions/695.

Let us know if this makes sense for your quick design update!
